python/cudf/cudf/core/column/lists.py

```diff
@@ -237,9 +237,10 @@ def contains(self, search_key):
         Series([False, True, True])
         dtype: bool
         """
+        search_key = cudf.Scalar(search_key)
```

Comment:
I think this is the only thing we were missing. Can we keep the Cython as taking a `DeviceScalar` with this change?